mk/db.mk

```diff
@@ -12,7 +12,7 @@ DATABASE_COMPOSE_OPTIONS=CONTENT_DATABASE_USER=$(DATABASE_USER) \
 
 .PHONY: .db-health-wait
 .db-health-wait:
-	@while [ $($(COMPOSE_COMMAND) ps | grep postgres-content | grep healthy) 2> /dev/null ]; do printf "."; sleep 1; done
+	@while [ "$$($(DOCKER) ps | grep postgres-content | grep healthy | wc -l)" != "1" ]; do printf "."; sleep 1; done
```

Comment:
What is the significance of this change?

Reply:
i mentioned this in the commit message, it fixes the 'make compose-clean; make compose-up;` not handling db health correctly.  I was doing that a lot as part of this to test, so i figured I'd go ahead and fix it.  Happy to split it out to another pr if desired.

Reply:
🥚Cellent!

Reply:
```suggestion
	@while [ $$($(DOCKER) ps | grep postgres-content | grep healthy | wc -l) != "1" ]; do printf "."; sleep 1; done
```

Reply:
I think the above fixes this?

Reply:
updated to remove the quotes from both